### DISCUSSION TITLE
Can this be used to dump a SQLite DB and Import into a MySQL DB?

### DISCUSSION THREAD

**Discussion by warwickchapman** [amacneil/dbmate#240] [Support] [answered]
I cannot see how to export the data, only the schema.

I'm looking for a literal migration from one DBMS to another.

**Comment by amacneil** [comment, marked as answer]
No, for export you should check out [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) or sqlite [.dump command](https://www.sqlitetutorial.net/sqlite-dump/).

For import just run the generated sql file against the new database.

**Reply by warwickchapman** [reply]
Not that simple. Not by a long way. I couldn’t find a tool. I ended up editing all the SQL vim to make it compatible for MySQL import.
